src/styles/Button.js

```diff
@@ -0,0 +1,37 @@
+import Button from 'react-bootstrap/Button';
+import css from './Button.module.scss';
+import { Container} from 'react-bootstrap';
+import ProductAll from './ProductAll';
+import ProductCard from './ProductCard';
+
+function OutlineTypesExample() {
+  return (
+    <>
+    <div className={css.wrap}>
+      <Button variant="outline-primary">인기</Button>{' '}
+      <Button variant="outline-primary">최신</Button>{' '}
+      <Button variant="outline-primary">팔로잉</Button>{' '}
+     
+    </div>
+    <Container>
+        <div className={css.Container}>
```

Comment:
className 컨벤션 지켜주시면 감사하겠습니다!
```suggestion
        <div className={css.container}>
```